templates/appliance/adguard/intel-nuc.html

```diff
@@ -10,6 +10,6 @@
 
 {% include 'appliance/shared/_intel-nuc.html' %}
 
-{% include 'appliance/adguard/_start-using.html' %}
+{% include 'appliance/'+appliance.url_name+'/_next-steps.html' %}
```

Comment:
We might want to remove the _ on templates to sace issues in the future.

Reply:
@anthonydillon the idea is that we distinguish explicitly between include files that are internal to the template system and publicly exposed template files. This is to stop people just inspecting our template files through the website. This `_next-steps.html` file is an internal include file right? So it should explicitly *not* have the underscore removed.

What we've done with `navigation-h ...` files is pretty hacky, but you have to think about it as us deliberately creating a public page containing some HTML for JavaScript to consume, not as a template include file, which it's not.

Reply:
This also follows the pattern used in Jekyll and Eleventy and other static site systems, where `_` files are removed, and includes are kept in an `_includes` folder. I think this is much neater and what we should try to replicate.